### PR TITLE
Add credit consumption to cost per query

### PR DESCRIPTION
**Pull request by yingyingqiqi**
Processing [# 65](https://github.com/get-select/dbt-snowflake-monitoring/issues/65)

### CODE REVIEW

**Review by ian-whitestone — APPROVED** [review]
Thanks @yingyingqiqi !

File: models/cost_per_query.yml [hunk 1]
```diff
@@ -18,10 +18,16 @@ models:
         description: When the query began executing on the warehouse (in the UTC time zone). This will always be after the start_time.
       - name: compute_cost
         description: Compute costs associated with the query, in the primary currency of your account. Can be 0 if the query did not run on a warehouse.
+      - name: compute_credits
+        description: Compute credits associated with the query, in the primary currency of your account. Can be 0 if the query did not run on a warehouse.
       - name: cloud_services_cost
         description: Cloud service costs associated with the query, in the primary currency of your account. Can be 0 if total cloud services credits consumption was less than 10% of total compute credits consumption on that day.
+      - name: cloud_services_credits
+        description: Cloud service credits associated with the query, in the primary currency of your account. Can be 0 if total cloud services credits consumption was less than 10% of total compute credits consumption on that day.
```
Comment (NiallRees) [inline models/cost_per_query.yml:26]:
```suggestion
        description: Cloud service credits associated with the query. Can be 0 if total cloud services credits consumption was less than 10% of total compute credits consumption on that day.
```

File: models/cost_per_query.yml [hunk 2]
```diff
@@ -18,10 +18,16 @@ models:
...
       - name: query_cost
         description: Total cost associated with the query, calculated as sum of compute_cost and cloud_services_cost, in the primary currency of your account.
+      - name: query_credits
+        description: Total cost associated with the query, calculated as sum of compute_credits and cloud_services_credits, in the primary currency of your account.
```
Comment (NiallRees) [inline models/cost_per_query.yml:30]:
```suggestion
        description: Total credits associated with the query, calculated as sum of compute_credits and cloud_services_credits.
```

File: models/query_history_enriched.yml [hunk 3]
```diff
@@ -156,10 +156,16 @@ models:
         description: When the query began executing on the warehouse (in the UTC time zone). This will always be after the start_time.
       - name: compute_cost
         description: Compute costs associated with the query, in the primary currency of your account. Can be 0 if the query did not run on a warehouse.
+      - name: compute_credits
+        description: Compute credits associated with the query, in the primary currency of your account. Can be 0 if the query did not run on a warehouse.
```
Comment (NiallRees) [inline models/query_history_enriched.yml:160]:
```suggestion
        description: Compute credits associated with the query. Can be 0 if the query did not run on a warehouse.
```

File: models/query_history_enriched.yml [hunk 4]
```diff
@@ -156,10 +156,16 @@ models:
...
       - name: cloud_services_cost
         description: Cloud service costs associated with the query, in the primary currency of your account. Can be 0 if total cloud services credits consumption was less than 10% of total compute credits consumption on that day.
+      - name: cloud_services_credits
+        description: Cloud service credits associated with the query, in the primary currency of your account. Can be 0 if total cloud services credits consumption was less than 10% of total compute credits consumption on that day.
       - name: query_cost
         description: Total cost associated with the query, calculated as sum of compute_cost and cloud_services_cost, in the primary currency of your account.
+      - name: query_credits
+        description: Total credits associated with the query, calculated as sum of compute_credits and cloud_services_credits, in the primary currency of your account.
```
Comment (NiallRees) [inline models/query_history_enriched.yml:168]:
```suggestion
        description: Total credits associated with the query, calculated as sum of compute_credits and cloud_services_credits.
```

**Pull request merged**